dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -230,4 +230,14 @@ def create_admin_script
     get :show, id: admin_script.name
     assert_response :success
   end
+
+  test "should have one game group if one stage" do
```

Comment:
this seems like it goes with something else? But if it passes feel free to check it in here :)